src/Saritasa.NetForge.UseCases/Services/EntityService.cs

```diff
@@ -413,4 +414,12 @@ public Task DeleteEntityAsync(object entity, Type entityType, CancellationToken
     {
         return dataService.BulkDeleteAsync(entities, entityType, cancellationToken);
     }
+
+    /// <inheritdoc />
+    public bool ValidateEntity(object instance, ref List<ValidationResult> errors)
+    {
+        var context = new ValidationContext(instance, serviceProvider, null);
+
+        return Validator.TryValidateObject(instance, context, errors, true);
```

Comment:
Prefer specifying parameter name for `null` and `bool`
```suggestion
        var context = new ValidationContext(instance, serviceProvider, items: null);

        return Validator.TryValidateObject(instance, context, errors, validateAllProperties: true);
```